articles/multifactor-authentication/index2.md

```diff
@@ -15,49 +15,45 @@ Multifactor Authentication (MFA) is a method of verifying a user's identity by r
 
 Auth0 supports the following methods of implementing MFA:
```

Comment:
Thanks! 
Would you be interested in adding my other changes in PR #1466 https://github.com/auth0/docs/pull/1466 (mostly just a warning panel linking to a new doc about Resource Owner, and linking with doc titles, and a description) into this PR? 
Or I can do it after this is merged.